Gopkg.toml

```diff
@@ -88,9 +88,9 @@ required = [
   name = "github.com/giantswarm/micrologger"
   branch = "master"
 
-[[constraint]]
+[[override]]
+  branch = "cluster-controller"
```

Comment:
We don't want to merge this PR yet. So if we leave this on master we should be able to merge these changes now. 

Otherwise we can leave this open until https://github.com/giantswarm/operatorkit/pull/338 is merged.

Reply:
@rossf7 I think most PRs for the checklist have this change. For now, I will just test on testing installation and waiting for prior PR merged on operatorkit. WDYT?

Reply:
Nevermind, I will revert this one and would make this PR as ready.

Reply:
Thanks I think that is easier. Better to be able to merge sooner.